glooshot/docs/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Gloo Shot
+weight: 99
+---
+
+#### Service Mesh Chaos Engineering
+
+Gloo Shot is a chaos engineering framework for service meshes.
+
+
+## Using Gloo Shot
+- **Harden your mesh**: Gloo Shot allows you to test failure modes before they occur in production.
+- **Preview architectural changes**: Real deployments have different performance characteristics than your production environment. Gloo Shot allows you to simulate your productionn environment (latency, faults, etc.) prior to deployment.
+
+### Getting started
+
+- Gloo Shot is easy to [install](glooshot.solo.io/installation/install/) from the `glooshot` command line tool.
```

Comment:
So I guess this will work on the live site, but they don't properly link locally. We should use ref links for Hugo... something like 

`[Rate-limiting service]({{< ref "/enterprise/ratelimit.md">}})`

Reply:
Take a look here for more: https://gohugo.io/content-management/cross-references/

Reply:
ah, good catch